webcompat/static/js/lib/models/issue.js

```diff
@@ -91,13 +91,11 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var el = body;
-    el = $("strong:contains('Description')", el);
-    if (el[0]) {
-      var description = el[0].nextSibling.textContent;
-      description = description.replace(": ", "");
-      return description != null ? description.slice(0, 74) : null;
-    }
+    var div = document.createElement("div");
+    div.innerHTML = body;
+    var regex = /Description:(.*)+\n/gm;
```

Comment:
I don't think my questions about the `gm` flags were answered from the first review. I don't think we need them (unless I misunderstand something?) https://github.com/webcompat/webcompat.com/pull/2805#discussion_r257047185

Reply:
And there is probably a typo in there. `+\n` instead of `\n+`

My original suggestion was around, but if we stop after the first `\n`, but we could just probably do:

`/Description:(.+)\n/`